frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -622,13 +636,25 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
         updateAnimation: () => {
             // The main loop of the player. Called on each frame
             const rrwebPlayerTime = values.player?.replayer?.getCurrentTime()
-            let newTimestamp = values.fromRRwebPlayerTime(rrwebPlayerTime)
+            let newTimestamp = values.fromRRWebPlayerTime(rrwebPlayerTime)
+
+            const skip = values.playerSpeed * (1000 / 60) // rough animation fps
+            if (
```

Comment:
and then if the timestamp has got stuck we skip forward a frame

Reply:
Does this work without calling `replayer?.play` and if so can we remove it?
The newTimestamp will trigger the codepaths that will eventually call `play` which feels safer as there are a lot of guards to ensure we don't play if the timestamp is outside the range of the current window for example